kube_apiserver_metrics/README.md

```diff
@@ -26,10 +26,10 @@ annotations:
     '[{ "prometheus_url": "https://%%host%%:%%port%%/metrics", "bearer_token_auth": "true" }]'
 ```
 
-Then the Datadog Cluster Agent schedules the check(s) for each endpoint onto Datadog Agent(s).
+Then the Datadog Cluster Agent schedules the check(s) for each endpoint onto Datadog Agent(s). 
 
 You can also run the check by configuring the endpoints directly in the `kube_apiserver_metrics.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][4].
-See the [sample kube_apiserver_metrics.d/conf.yaml][2] for all available configuration options.
+You must add `cluster_check:true` to your [configuration file][9] when using a ConfigMap to configure cluster checks. See the [sample kube_apiserver_metrics.d/conf.yaml][2] for all available configuration options.
```

Comment:
Thanks! Two more small suggestions:
- `cluster_check:true` -> `cluster_check: true` (added space for consistency in other docs)
- "when using a ConfigMap" -> "when using a static configuration file or ConfigMap"

cc @ahmed-mez in case you have any suggestions/improvements